docs/source/usage/python.rst

```diff
@@ -576,6 +576,15 @@ This module provides elements for the accelerator lattice.
    :param rc: Radius of curvature in m.
    :param nslice: number of slices used for the application of space charge
 
+.. py:class:: impactx.elements.ExactSbend(ds, phi, B, nslice=1)
+
+   An ideal sector bend using the exact nonlinear map.
```

Comment:
Discussed:
- hard edge approximation
- assumption of normal entry & exit to the element